resources/state_table/huntsman.yaml

```diff
@@ -49,6 +51,14 @@ transitions:
         source: parked
         dest: housekeeping
         trigger: clean_up
+    -
+        source: parked
+        dest: taking_darks
+        trigger: set_park
```

Comment:
I think the trigger to enter the `taking_darks` state should have its own trigger. Something like `trigger: take_darks`.